### PR TITLE
Fix for issue #3210

### LINKED ISSUE #3210: Crash when uploading a picture via Nearby
API level: 28
Android version: 9
Device manufacturer: samsung
Device model: SM-G970F
Device: beyond0
Network type: wifi
App version name: 2.11.0.493 (GitHub commit 8ccad22)

Steps to reproduce:

1. Start Commons app
2. Tap "Nearby"
3. Select any pin
4. Tap the + button
5. Tap gallery
6. Select any picture
7. Crash

Same if tapping camera at step 5

### PR DESCRIPTION
**Pull request by IgnacioGarcia198**
**Description (required)**
When the place is not null but the descriptions list is empty there was a crash on getting the first element of the Descriptions list. 
Fixes #3210  Crash when uploading a picture via Nearby
I just check if the list is empty, and in that case I add a new Description to the list.
What changes did you make and why?

**Tests performed (required)**

Tested betaDebug on Asus ASUS_X00ID with API level 25.

**Screenshots showing what changed (optional - for UI changes)**

Need help? See https://support.google.com/android/answer/9075928

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._
Screenshot_20191119-194317

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:3.5.2'
```
Comment (neslihanturan) [inline build.gradle]: Please revert these extra irrelevant changes. You should ignore Android Studio's warnings.

**Comment by neslihanturan** [comment]
Hey @IgnacioGarcia198 , thanks a lot for fast responses. I requested some changes about irrelevant changes. Plus please make sure you followed the PR guideline, ie. title format and description. So that reviewers can understand what this PR does easier.

File: app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java [hunk 2]
```diff
@@ -142,6 +142,9 @@ private UploadItem getUploadItem(UploadableFile uploadableFile,
                 createdTimestampSource);
         if (place != null) {
             uploadItem.title.setTitleText(place.name);
+            if(uploadItem.descriptions.isEmpty()) {
```
Comment (ashishkumar468) [inline app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java]: Should we rather just not update the descriptions if they are empty, instead of creating a new object ?

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Thu Mar 14 14:54:52 EDT 2019
```
Comment (ashishkumar468) [inline gradle/wrapper/gradle-wrapper.properties]: This is not related to this PR, Please revert this file

File: app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java (hunk 2 above)
Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java]: And another question can be is it normal that descriptions are empty at this state?

File: build.gradle (hunk 1 above)
Comment (ashishkumar468) [reply build.gradle]: As @neslihanturan said, if we want to update the Gradle version, we should create a separate  PR/issue for the same.
